examples/machinedeployment/machinedeployment.yaml

```diff
@@ -73,13 +72,15 @@ spec:
         permissions: "0600"
       joinConfiguration:
         nodeRegistration:
+          name: '{{ local_hostname }}'
+          criSocket: "/var/run/containerd/containerd.sock"
```

Comment:
You shouldn't need this here (unless you are attempting to support k8s v1.13), since kubeadm should autodetect the criSocket for you by default.